upgrading-open-data-hub.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: upgrade
+
+[id="upgrading-open-data-hub_{context}"]
+= Upgrading Open Data Hub
+
+[role='_abstract']
+You can upgrade the Open Data Hub Operator from version 1 to version 2 from the OpenShift web console. 
```

Comment:
To avoid the double occurrence of "from" here, I suggest changing to "using the OpenShift web console."